org.frescobaldi.Frescobaldi.yaml

```diff
@@ -27,16 +27,10 @@ finish-args:
   - --socket=x11
```

Comment:
you need to use `fallback-x11` here since you enabled wayland

Reply:
Thanks, fixed it